packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -24,6 +24,7 @@ import { Link } from '@bbc/psammead-story-promo';
 import { oneOf, shape, string } from 'prop-types';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { formatDuration } from '@bbc/psammead-timestamp-container/src/utilities/timestampUtilities';
```

Comment:
Can you just use `@bbc/psammead-timestamp-container/utilities`?

Reply:
You will be able to do what @tochwill suggests, after this PR https://github.com/bbc/psammead/pull/3085 gets merged

Reply:
Oops, thought it was already merged! I'll take a look